common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -319,6 +315,39 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
         return true;
     }
 
+    public boolean containsKey(Object key) {
+        return containsKey(objectToKey(key));
+    }
+
+    @Override
+    public V get(Object key) {
+        return get(objectToKey(key));
+    }
+
+    @Override
+    public V put(@O@ key, V value) {
+        return put(objectToKey(key), value);
+    }
+
+    @Override
+    public V remove(Object key) {
+        return remove(objectToKey(key));
+    }
+
+    @Override
+    public Set<@O@> keySet() {
+        return keySet;
+    }
+
+    @Override
+    public Set<Entry<@O@, V>> entrySet() {
+        return entrySet;
+    }
+
+    private @k@ objectToKey(Object key) {
+        return (@k@) ((@O@) key).@KEY_NUMBER_METHOD@();
```

Comment:
I had previously had a check for `instanceof`, but @buchgr asked that it be removed for performance.  I don't think we care since it will throw an exception one way or another.